superduperdb/base/logger.py

```diff
@@ -15,7 +16,16 @@
 
 
 class Logging:
-    if CFG.logging.type == LogType.LOKI:  # Send logs to Loki
+    if CFG.logging.type == LogType.Database:  # Send logs to Database
```

Comment:
This won't work -- the idea is send the logs of the workers only to the database.

Reply:
Use case: "User would like to see what's going on in his jobs from a Jupyter notebook".